examples/compression-zstd/main.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"log"
+	"time"
+
+	zstd_middleware "github.com/momentohq/client-sdk-go-compression-zstd/zstd_compression"
```

Comment:
```suggestion
	zstdMiddleware "github.com/momentohq/client-sdk-go-compression-zstd/zstd_compression"
```
**I'm** not picky-- it's my IDE :-)

Reply:
ah dang, my IDE is not as opinionated apparently haha
will open another small PR for this (and any other feedback!)